### PR TITLE
CET: add Glibc not supported judgement for SHSTK enabled by Glibc code

### PR DESCRIPTION
**Pull request by xupengfe**
Signed-off-by: Pengfei Xu <pengfei.xu@intel.com>

### CODE REVIEW

File: cet/shstk_alloc.c [hunk 1]
```diff
@@ -80,6 +80,7 @@ int main(int argc, char *argv[])
 
 	if (!_get_ssp()) {
```
Comment (ysun) [inline cet/shstk_alloc.c:81]: Does it need to update here by using the new func `get_ssp`?

File: cet/quick_test.c [hunk 2]
```diff
@@ -74,10 +81,9 @@ static void user2_handler(int signum, siginfo_t *si, void *uc)
 int main(int argc, char *argv[])
 {
 	struct sigaction sa;
-	int ret;
-
-	int cpu;
+	int ret, cpu;
 	cpu_set_t set;
+	unsigned long ssp;
```
[the remark this answers is not in the document]
Comment (xupengfe) [reply cet/quick_test.c:86]: Yes, will correct it.

Thanks!

File: cet/shstk_alloc.c (hunk 1 above)
Comment (xupengfe) [reply cet/shstk_alloc.c:81]: Yes, it's better.

Thanks!

File: cet/shstk_alloc.c [hunk 3]
```diff
@@ -28,6 +28,13 @@
 
 size_t shstk_size = 0x200000;
 
+#define get_ssp()						\
+({								\
+	unsigned long _ret;						\
```
Comment (ysun) [inline cet/shstk_alloc.c:33]: It should be better if the '\\' is aligned.

File: cet/quick_test.c [hunk 4]
```diff
@@ -13,6 +13,13 @@
 #include <sys/wait.h>
 #include <sched.h>
 
+#define get_ssp()						\
+({								\
+	unsigned long _ret;						\
```
Comment (ysun) [inline cet/quick_test.c:18]: It should be better if the '\\' is aligned.

File: cet/shstk_alloc.c (hunk 3 above)
Comment (xupengfe) [reply cet/shstk_alloc.c]: Yes, done.

Thanks!

File: cet/quick_test.c (hunk 4 above)
Comment (xupengfe) [reply cet/quick_test.c]: Yes, done.

Thanks!

**Pull request merged**